scripts/make/qemu.mk

```diff
@@ -30,9 +30,19 @@ qemu_args-$(FS) += \
   -device virtio-blk-$(vdev-suffix),drive=disk0 \
   -drive id=disk0,if=none,format=raw,file=$(DISK_IMG)
 
-qemu_args-$(NET) += \
-  -device virtio-net-$(vdev-suffix),netdev=net0 \
-  -netdev user,id=net0,hostfwd=tcp::5555-:5555,hostfwd=udp::5555-:5555
+qemu_args-$(NET) += -device virtio-net-$(vdev-suffix),netdev=net0
+ifeq ($(NETDEV), user)
+  qemu_args-$(NET) += \
+    -netdev user,id=net0,hostfwd=tcp::5555-:5555,hostfwd=udp::5555-:5555
+else ifeq ($(NETDEV), tap)
+  qemu_args-$(NET) += \
+    -netdev tap,id=net0,ifname=qemu-tap0,script=no,downscript=no
+endif
+
+ifeq ($(MODE), debug)
+  qemu_args-$(NET) += \
```

Comment:
`qemu_args-debug` is used when `MODE=debug`

Reply:
`qemu_args-debug` is not used when `MODE=debug`. It is used for gdb debugging when run with `make debug`.

Here I intend to capture packets when `MODE=debug`, regardless of whether gdb is being used or not.